Parsers/ASimAuthentication/Parsers/ASimAuthenticationOktaOSS.yaml

```diff
@@ -13,7 +13,7 @@ References:
   - Title: ASIM
     Link: https:/aka.ms/AboutASIM
 Description: |
-  This ASIM parser supports normalizing Okta sign in logs, stored in the Okta_CL table and in OktaV2_CL table, to the ASIM Authentication schema. 
+  This ASIM parser supports normalizing Okta sign in logs, stored in the Okta_CL table, to the ASIM Authentication schema. 
```

Comment:
do we need emptyOctaV2Table data table at line 29 since this parser doesn't include v2

Reply:
Done